Ginger/GingerCoreNET/ValueExpressionLib/ValueExpression.cs

```diff
@@ -375,6 +374,39 @@ public enum eFlowDetailsObjects
             }
         }
 
+        public static Mockdata GetMockDataDatasetsFunction(string MockDataExpression)
+        {
+            try
+            {
+                string objStr = string.Empty;
```

Comment:
Seems this code is being duplicated mock data editor page, please try to avoid duplicate code

Reply:
done